server/libs/domain/src/media/media.repository.ts

```diff
@@ -47,5 +47,5 @@ export interface IMediaRepository {
   // video
   extractVideoThumbnail(input: string, output: string, size: number): Promise<void>;
   probe(input: string): Promise<VideoInfo>;
-  transcode(input: string, output: string, options: any): Promise<void>;
+  transcode(input: string, output: string, options: any, twoPass: boolean): Promise<void>;
```

Comment:
Not critical, but I don't like having too many arguments in an interface. Everything else uses input/output + options I believe. Could we merge the last to into something like this? `options: TranscodeOptions`

Eg:
```typescript
export interface TranscodeOptions {
  outputOptions: string[];
  twoPass: boolean;
}


```

Reply:
Good call, added